README.md

```diff
@@ -1,30 +1,27 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Newsweek Milestone 1
 
-> One paragraph statement about the project.
+> Building Newsweek's page clone using CSS framework (Bootstrap) and media quesries
 
-![screenshot](./app_screenshot.png)
-
-Additional description about the project and its features.
+![Desktop](./img/desktop.png)
+![Tablet](./img/tablet.png)
+![Mobile](./img/mobile.png)
 
```

Comment:
## Improve the README
- [ ] I would suggest we have a single screenshot of the page. The rest of the breakpoints can get showcased in a nice and simple explanation. what are your thoughts on this?